en: add a simpe desc about HA (#366)
pingcap/docs-dm: en/overview.md

**What's new in DM v2.0:**

> - [High availability of data migration task](overview.md#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.

Reviewer: ```suggestion
- [High availability of data migration task](#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.
```

- Handling the errors during data migration tasks
- Verifying the configuration correctness of data migration tasks

### High availability

> When multiple DM-master nodes are deployed, all DM-master nodes will use the embedded etcd to form a cluster. The DM-master cluster is used to store metadata such as cluster node information and task configuration. The leader node elected through etcd is used to provide services related to cluster management and data migration task management. Therefore, if the number of available DM-master nodes exceeds half of the deployed nodes, services can be provided normally.

Reviewer: ```suggestion
When you deploy multiple DM-master nodes, all DM-master nodes use the embedded etcd to form a cluster. The DM-master cluster is used to store metadata such as cluster node information and task configuration. The leader node elected through etcd is used to provide services such as cluster management and data migration task management. Therefore, if the number of available DM-master nodes exceeds half of the deployed nodes, the DM cluster can normally provide services.
```
Use active voice and present tense

> When the number of deployed DM-worker nodes exceeds the number of upstream MySQL/MariaDB nodes, the related DM-worker nodes that exceed the number of upstream nodes will be idle by default. If a DM-worker node goes offline or is isolated from the DM-master leader, the DM-master can automatically schedule data migration tasks of the original DM-worker node to other idle DM-worker nodes. (If the worker node is isolated, the related data migration tasks will be automatically stopped); if no idle DM-worker nodes can be scheduled, the data migration tasks of the original DM-worker will not be performed.

Reviewer: ```suggestion
When the number of deployed DM-worker nodes exceeds the number of upstream MySQL/MariaDB nodes, the extra DM-worker nodes are idle by default. If a DM-worker node goes offline or is isolated from the DM-master leader, DM-master automatically schedules data migration tasks of the original DM-worker node to other idle DM-worker nodes. (If a DM-worker node is isolated, it automatically stops the data migration tasks on it); if there are no available idle DM-worker nodes, the data migration tasks of the original DM-worker cannot be performed.
```

Reviewer: PTAL at my suggestion, especially this one: `If a DM-worker node is isolated, it automatically stops the data migration tasks on it` @csuzhangxc

Reviewer 2: your suggestion is right. 👍
